.github/workflows/release.yml

```
@@ -86,10 +86,36 @@ jobs:
             export PATH="$(nix build github:runtimeverification/kup --no-link --json | jq -r '.[].outputs | to_entries[].value')/bin:$PATH"
             kup publish --verbose k-framework-binary .#k --keep-days 180
             kup publish --verbose k-framework-binary .#k.openssl.procps.secp256k1 --keep-days 180
-      
+
+  test-package-kframework-wheel:
```

For consistency in naming with other `pyk`-related jobs (`pyk-build-docs`, `pyk-publish`), consider

```suggestion
  pyk-build-wheel:
```

---

Also let's add something like `timeout-minutes: 5`, just in case.

---

Good idea! I added both suggestions.